GlowSequencer/Usb/ClubConnectionUtility.cs

```diff
@@ -260,6 +260,36 @@ public void Start(string connectedPortId)
             }
         }
 
+        public void StartSync(IEnumerable<string> connectedPortIds)
+        {
+            List<UsbDevice> devices = new List<UsbDevice>();
+            try
+            {
+                foreach (string connectedPortId in connectedPortIds)
+                {
+                    UsbDevice device = OpenDevice(connectedPortId);
+                    devices.Add(device);
+                }
+
+                foreach (UsbDevice device in devices)
+                {
+                    StartProgramOnDevice(device);
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw;
+            }
```

Comment:
note: no need to do this when rethrowing unmodified, just let the exception bubble up and let the caller decide.

also avoid catching generic Exception since it hides unnoticed programmer errors that are not `UsbOperationException`s.

just FYI :)